src/inc-lib-init.php

```diff
@@ -1516,9 +1516,14 @@ function lovd_getVariantInfo ($sVariant, $sTranscriptID = '', $bCheckHGVS = fals
     }
 
     if (!$aVariant['earliest_end'] && $aVariant['latest_start']) {
-        // We now know we are dealing with a case such as g.(1_5)ins. This means that the positions
-        //  are uncertain, but somewhere within the range as given within the parentheses. We add a
-        //  message to make sure users know our interpretation and can make sure they meant it as such.
+        // We now know we are dealing with a case such as g.(1_5)ins. This means
+        //  that the positions are uncertain, but somewhere within the range as
+        //  given within the parentheses. We add a message to make sure users
+        //  know our interpretation and can make sure they meant it as such.
+        // Note that IPOSITIONRANGE and IUNCERTAINPOSITIONS both send the same
+        //  message to the user about uncertain positions. However, internally,
+        //  this notice is used to determine whether the variant needs a suffix
+        //  because the variant's position is a single, uncertain range.
```

Comment:
I decided in the end not to merge `IPOSITIONRANGE` and `IUNCERTAINPOSITIONS`, it wouldn't make the code more readable.